src/Microsoft.Dnx.Compilation.CSharp/RoslynProjectReference.cs

```diff
@@ -312,6 +314,11 @@ private EmitOptions GetEmitOptions(out bool emitPdb)
                 references: CompilationContext.Compilation.References,
                 options: compilationOptions);
 
+            compilation = compilation.AddSyntaxTrees(new[]
```

Comment:
Yes since I am using `assemblyName.CultureName` that should never be null. If no value is specified it defaults to `neutral`. Should I add that check still just in case?